src/Evolution/Systems/ForceFree/Tags.hpp

```diff
@@ -115,6 +115,22 @@ struct TildeJ : db::SimpleTag {
 struct LargestCharacteristicSpeed : db::SimpleTag {
   using type = double;
 };
+
+/*!
+ * \brief A scalar variable used for masking interior of neutron star(s) when
+ * running neutron star magnetosphere simulations.
+ *
+ * For each grid points, we assign the value +1.0 if the grid point is located
+ * outside the NS and assign -1.0 if it's located inside the NS.
+ *
+ * \note If initial data is not simulating neutron star magnetosphere, all
```

Comment:
Would it make sense to have this be a `std::optional` so that it's easy to detect if _all_ points are `+1` because then the optional is null?